kafka/4.2/modules/ROOT/pages/kafka-connector-design-center.adoc

```diff
@@ -0,0 +1,217 @@
+= Use Flow Designer to Configure Apache Kafka Connector - Mule 4
+
```

Comment:
You removed a page alias for this file in the 4.1 version. If you remove something from an earlier version, you'll need to make sure you add it to the current version.

Reply:
I thought you said we only needed these for the Reference and index topics.